### ISSUE TITLE
Provide a set of OOTB fast-icons

### ISSUE THREAD

**Issue by ameyasubhedar** [microsoft/fast#4842]
# 🙋 Feature Request

For the completeness of the adaptive UI/UX foundation system. What are the plans of providing the fast-icons ?

## 🤔 Expected Behavior

A set of high quality icons in all standard supported formats.

## 😯 Current Behavior

No OOTB icons are available.

**Comment by EisenbergEffect** [comment]
Ping @bheston

**Comment by chrisdholt** [comment]
I don't think we currently have plans for creating and supporting an icon set. While it's possible we may pick this up in the future, my perspective here as a maintainer is that it doesn't currently align with our goals and priorities for the project. While we don't handle *everything*, FAST can be implemented to support any variation of high quality icon set available from any producer across the web. My recommendation would be to leverage SVG's as they are a standardized element and approach which we offer some first class support for slotting into controls (width, height, fill).

I'm happy to  leave this in the backlog, but I don't currently see it as something we'd create and maintain at this point in time. With that said, it would be awesome to see the community pick up and create an "Adaptive Iconography" using FAST and who knows, we may pick it up in the future at some point :).

**Comment by bheston** [comment]
Agreed with Chris regarding using SVG icons. We've updated all of the components to allow SVG of any size to be used and the components should accommodate them without cropping or resizing the icons. For instance, the [Fluent System Icons](https://github.com/microsoft/fluentui-system-icons) work great, as do any of the other SVG icon libraries.

We have discussed icons in Adaptive UI before, and the biggest real problem we've encountered in that area is the need to scale icons. The Fluent set I mentioned above comes in many sizes to account for that, but something I always thought would be great was the ability to logically scale a mono-stroke SVG icon by moving the vector points without increasing the line width. That would allow you to scale a 16x16 icon up to something like 20px or 24px if you type size increased, but if you do that by scaling alone you end up with blurry lines. Other aspects we've considered are rounding amount, thickness, and overlap with icon animation (as part of Adaptive Motion).
